pkg/cmd/projectgroup/view/view.go

```diff
@@ -73,37 +78,32 @@ func NewCmdView(f factory.Factory) *cobra.Command {
 	return cmd
 }
 
-func viewRun(opts *ViewOptions) error {
+func viewRun(opts *ViewOptions, cmd *cobra.Command) error {
 	projectGroup, err := opts.Client.ProjectGroups.GetByIDOrName(opts.idOrName)
 	if err != nil {
 		return err
 	}
 
-	fmt.Fprintf(opts.out, "%s %s\n", output.Bold(projectGroup.GetName()), output.Dimf("(%s)", projectGroup.GetID()))
-
-	if projectGroup.Description == "" {
-		fmt.Fprintln(opts.out, output.Dim(constants.NoDescription))
-	} else {
-		fmt.Fprintln(opts.out, output.Dim(projectGroup.Description))
-	}
-
-	projects, err := opts.Client.ProjectGroups.GetProjects(projectGroup)
+	allProjects, err := opts.Client.ProjectGroups.GetProjects(projectGroup)
 	if err != nil {
 		return err
 	}
-	fmt.Fprintf(opts.out, output.Cyan("\nProjects:\n"))
-	for _, project := range projects {
-		fmt.Fprintf(opts.out, "%s (%s)\n", output.Bold(project.GetName()), project.Slug)
-	}
 
-	url := fmt.Sprintf("%s/app#/%s/projects?projectGroupId=%s", opts.Host, projectGroup.SpaceID, projectGroup.GetID())
-
-	// footer
-	fmt.Fprintf(opts.out, "\nView this project group in Octopus Deploy: %s\n", output.Blue(url))
-
-	if opts.flags.Web.Value {
-		browser.OpenURL(url)
-	}
-
-	return nil
+	return output.PrintArray(allProjects, cmd, output.Mappers[*projects.Project]{
```

Comment:
For the basic output format, it should return what it did previously, otherwise we are losing some functionality. 

Example of the previous basic output.
```
Default Project Group (ProjectGroups-1)
No description provided

Projects:
NonVcs (nonvcs)
VCS (vcs)

View this project group in Octopus Deploy: http://localhost:8066/app#/Spaces-1/projects?projectGroupId=ProjectGroups-1
```